src/db.rs

```diff
@@ -120,51 +114,53 @@ pub async fn list_available_hosts(db: &Client, cutoff: u64) -> Result<Vec<HostSu
         doc! {
             "$group": {
                 "_id": "$name",
-                "IP": {"$first": "$IP"},
-                "timestamp": {"$first": "$timestamp"},
-                "sshSuccess": {"$first": "$sshSuccess"},
                 "holoNetwork": {"$first": "$holoNetwork"},
                 "channel": {"$first": "$channel"},
                 "holoportModel": {"$first": "$holoportModel"},
-                "hostingInfo": {"$first": "$hostingInfo"},
-                "error": {"$first": "$error"},
+                "sshStatus": {"$first": "$sshStatus"},
+                "ztIp": {"$first": "$ztIp"},
+                "wanIp": {"$first": "$wanIp"},
+                "holoportIdBase36": {"$first": "$holoportIdBase36"},
```

Comment:
I thin it should be `holoportId`, not `holoportIdBase36`. Compare https://hackmd.io/e29-lKrVQrWauVb1nZBZqA?view#Technical-details

Reply:
Ah, ok! :laughing: - It seems we have conflicting data in the same document. I actually just changed it to this naming bc of how it's listed in the `host_statistics.holoport_status` model here: https://hackmd.io/e29-lKrVQrWauVb1nZBZqA?both#host_statisticsholoport_status

Changing it back now. :+1: